package.json

```diff
@@ -138,6 +143,7 @@
     "browserify-fs": "^1.0.0",
     "browserify-zlib": "^0.2.0",
     "bs58check": "^2.1.2",
+    "buffer": "^6.0.3",
```

Comment:
I don't believe this will be needed with polyfill plugin

Reply:
See https://github.com/shapeshift/web/pull/8924#discussion_r1982944416